metaflow/plugins/aip/aip_metaflow_step.py

```diff
@@ -335,7 +335,7 @@ def kfp_metaflow_step(
         not "METAFLOW_USER" in metaflow_configs_new
         or metaflow_configs_new["METAFLOW_USER"] is None
     ):
-        metaflow_configs_new["METAFLOW_USER"] = "kfp-user"
+        metaflow_configs_new["METAFLOW_USER"] = "aip-user"
```

Comment:
Hmm this might be somewhat problematic for Flow Triggering Flow. I presume we'd need to support both `kfp-user` and `aip-user` in the RBAC bindings for a while. Should be easy enough but we'd need to make changes in `kubeflow-k8s-profiles` and in ZAP to cater to this change I'd imagine.

Likely just needs a Jira ticket and should be fairly quick to do and worthwhile to clean up this minor naming debt.

Reply:
see https://gitlab.zgtools.net/analytics/artificial-intelligence/ai-platform/aip-infrastructure/kubeflow-k8s-profiles/-/merge_requests/902 !